e2e/e2e_test.go

```diff
@@ -88,6 +89,7 @@ func TestServer(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt
```

Comment:
is this also from the linter? looks weird...

Reply:
This is a fix for scopelint. This prevents variable shadowing.